### DISCUSSION TITLE
Is the esp32-s3 supported?

### DISCUSSION THREAD

**Discussion by bensuffolk** [someweisguy/esp_dmx#105] [Q&A] [answered]
I am about to embark on a new hardware design and wondered if I should be using the straight esp32 I use in my current hardware that works great with this library, or if I should use the newer s3.

**Comment by someweisguy** [comment, marked as answer]
The ESP32-S3 is supported! It is completely up to you which you would like to use. :)

**Reply by bensuffolk** [reply]
Thanks, I didn't want to design with the new chip just to find it wasn't supported.
